Library/Homebrew/cmd/update-report.rb

```diff
@@ -132,11 +132,15 @@ def output_update_report
         "git", "-C", HOMEBREW_REPOSITORY, "tag", "--list", "--sort=-version:refname", "*.*"
       ).lines.first.chomp
 
-      if old_tag.blank? || (new_tag == old_tag)
+      Settings.write "latesttag", new_tag if new_tag != old_tag
+
+      if new_tag == old_tag
         puts "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
```

Comment:
```suggestion
        ohai "Updated Homebrew from #{shorten_revision(initial_revision)} to #{shorten_revision(current_revision)}."
```

Reply:
I'm fine with the `ohai` here, but I'm not sure we need it. I think it's important to draw attention to when we're updating to a new tag, but I'm not sure developers care that much about the new revision they're getting each time they update in between tags